### ISSUE TITLE
Sample code needs tidying up

### ISSUE THREAD

**Issue by blockscoped** [rokucommunity/rooibos#27]
The sample code in `samples/nonEvalSample` doesn't work straight off the bat. Some files are incorrectly referenced with the wrong paths etc.

The directory structure is also slightly confusing and it's not clear right away which files are test specs and which are part of the framework.

**Comment by georgejecook** [comment]
Do you have any suggestions as to what you would've expected?

**Comment by blockscoped** [comment]
Apologies if this oversteps the bounds a bit.
I've made a fork and moved things about slightly in the sample apps. This was mostly so I could get a handle on what things are and what they do. I think the added structure makes things clearer. 
https://github.com/blockscoped/rooibos/tree/master/samples/nonEvalSample

I forked the preprocessor too. https://github.com/blockscoped/rooibosPreprocessor/blob/master/src/index.ts
It now just takes option flags. This seems more extensible than using arguments. 
The main reason for making changes here was so I could specify the output location of the generated map file to fit in with the directory structure I've proposed in my branch of rooibos.

I realise these are quite sudden breaking changes, but it would be good to get your thoughts.
